app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,85 @@
+#TASKS = ["Laundry", "Feed Babette", "Clean room", "Sleep"] 
+class TasksController < ApplicationController
+  
+  def index
+    @tasks = Task.all
+  end
+  
+  def show
+    task_id = params[:id].to_i
+    @task = Task.find_by(id: params[:id])
```

Comment:
Consider refactoring the above two lines to either

```ruby
    task_id = params[:id].to_i
    @task = Task.find_by(id: task_id)
```

or

```ruby
    @task = Task.find_by(id: params[:id])
```

in order to help efficiently use local variables